flexmeasures/cli/tests/test_data_add.py

```diff
@@ -121,22 +122,33 @@ def test_add_reporter(app, db, setup_dummy_data, reporter_config):
     runner = app.test_cli_runner()
 
     cli_input_params = {
-        "sensor-id": report_sensor_id,
-        "reporter-config": "reporter_config.json",
+        "config": "reporter_config.yaml",
+        "parameters": "parameters.json",
         "reporter": "PandasReporter",
         "start": "2023-04-10T00:00:00 00:00",
         "end": "2023-04-10T10:00:00 00:00",
         "output-file": "test.csv",
     }
 
+    parameters = dict(
+        input=[
+            dict(name="sensor_1", sensor=sensor1.id),
+            dict(name="sensor_2", sensor=sensor2.id),
+        ],
+        sensor=report_sensor_id,
```

Comment:
Could this please also mention the alternative formulation using `output=` in a comment?

Reply:
Also, I was expecting the `sensor` way of passing the output sensor to only be possible via CLI options, not as a `parameter` field. I don't think the `ParametersSchema` should allow both ways, but I'm okay with the CLI option allowing both ways. That means the CLI should add the sensor to the parameters before the `ParametersSchema` is called. If that isn't trivial to implement, please let me know before implementing.